packages/components/pie-modal/src/index.ts

```diff
@@ -164,6 +164,7 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
             <dialog
                 id="dialog"
                 class="c-modal"
+                data-test-id="pie-modal"
```

Comment:
Is this attribute something that all next components will have to implement?

Reply:
This is just an attribute that is only used for locating the element during tests, it's a pattern we're used to following but if it doesn't seem logical then I'm sure we're happy to discuss it going forwards